src/vsr/replica.zig

```diff
@@ -2312,7 +2312,7 @@ pub fn ReplicaType(
                             break;
                         }
                     }
-                } else unreachable;
+                } else return;
```

Comment:
No header from the start_view headers is within our prepare_max, we bail out and wait for a more up-to-date start_view message. Earlier, our start_view headers were guaranteed to contain at least one hook op (op_prepare_max from previous checkpoints), but now that is not guaranteed as a primary only maintains a pristine journal from op_checkpoint → self.op.